Zotero/Scenes/Detail/PDF/PDFCoordinator.swift

```diff
@@ -542,3 +583,30 @@ extension PDFCoordinator: PdfAnnotationsCoordinatorDelegate {
         self.navigationController?.present(navigationController, animated: true, completion: nil)
     }
 }
+
+extension PDFCoordinator: DetailCitationCoordinatorDelegate {
+    func showLocatorPicker(using presenter: UINavigationController, for values: [SinglePickerModel], selected: String, picked: @escaping (String) -> Void) {
+        guard let coordinator = parentCoordinator as? DetailCoordinator else { return }
+        coordinator.showLocatorPicker(using: presenter, for: values, selected: selected, picked: picked)
+    }
+    
+    func showCitationPreviewError(using presenter: UINavigationController, errorMessage: String) {
+        guard let coordinator = parentCoordinator as? DetailCoordinator else { return }
+        coordinator.showCitationPreviewError(using: presenter, errorMessage: errorMessage)
+    }
+    
+    func showMissingStyleError(using presenter: UINavigationController?) {
+        guard let coordinator = parentCoordinator as? DetailCoordinator else { return }
+        coordinator.showMissingStyleError(using: navigationController)
+    }
+}
+
+extension PDFCoordinator: DetailCopyBibliographyCoordinatorDelegate {
+    func overlay(for viewController: CopyBibliographyViewController, isHidden: Bool) {
+        guard isHidden else { return }
+        viewController.willMove(toParent: nil)
```

Comment:
Same as before, no need to have this implemented in coordinator. I'd keep this in view controller.

Reply:
Check https://github.com/zotero/zotero-ios/pull/827/commits/7b4c1a6a242b77c079e1ac995801d4ab7e30008f